config/kubernetes/staging/ingress.yml

```diff
@@ -71,6 +71,8 @@ metadata:
       SecRuleEngine On
       SecRequestBodyLimit 1048576
       SecRequestBodyNoFilesLimit 1048576
+      SecRule TIME_HOUR "!@pm 06 07 08 09 10 11 12 13 14 15 16 17 18 19 20" \
```

Comment:
Should this go to 21?

Reply:
Last request is 21:59:59, server is UTC so 20:59:59